src/components/Greeting.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from "react-redux";
+import { getGreetings } from "../redux/greetings/greetings";
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const greeting = useSelector((state) => state.greetings)
+
+  useEffect(() => {
+    dispatch(getGreetings());
+  }, []);
+
+  return (
+    <div>
+      <h1>{greeting.message}</h1>
+    </div>
+  );
+};
+
+export default Greeting
```

Comment:
Nice work implementing the front end and connecting the rails api. However, there are some errors you need to fix to make your project abide by the standard best practices.

- [ ]  When you open your app, you are met with this errors regarding the use of semi colon and single quotes, as shown below. :point_down: 

- [ ] There is also another error on the react hook useEffect missing a dependency. 

Kindly check and fix them. :+1: 

![Screenshot from 2023-03-04 07-28-57](https://user-images.githubusercontent.com/65233860/222882328-af51dd4b-d168-46f9-b335-ce48d82e07fe.png)